05/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">    
+    <title>Szablon strony RWD</title>
+    <link rel="stylesheet" href="./style/reset.css">    
+    <link rel="stylesheet" href="./style/base.css">
+    <link rel="stylesheet" href="./style/mobile.css">    
+    <link rel="stylesheet" media="(min-width:800px)" href="./style/desktop.css"> 
+</head>
+<body>
+    <header class="header row">
+        <div class="header__container container">
+            <h1 class="header__logo cell"></h1>
+            <nav class="header__nav nav"> 
+                <button class="nav__hamburger" ></button>
```

Comment:
Kliknięcie w X na mobile nie powoduje zamknięcia, a wydaje się, że powinno to tak działać :)